src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
         "      )\n"
         "    ],\n"
         "\n"
-        "    continueCommand = .myCommand\n"
+        "    continueCommand = .myCommand,\n"
         "    # This is the command called to start your app back up after it has been\n"
         "    # shut down for inactivity. Here we're using the same command as for\n"
         "    # starting a new instance, but you could use different commands for each\n"
         "    # case.\n"
+        "\n"
+        "    metadata = (\n"
+        "      # Data which is not needed specifically to execute the app, but is useful\n"
+        "      # for purposes like marketing and display.  These fields are documented at\n"
+        "      # https://docs.sandstorm.io/en/latest/developing/publishing-apps/#add-required-metadata\n"
+        "      # and in the source code in the Metadata section of\n"
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
+        "      icons = (\n"
+        "        # Various icons to represent the app in various contexts.\n"
+        "        appGrid = (svg = embed \"path/to/128x128.svg\"),\n"
+        "        grain = (svg = embed \"path/to/24x24.svg\"),\n"
+        "        market = (svg = embed \"path/to/150x150.svg\"),\n"
+        "        marketBig = (svg = embed \"path/to/300x300.svg\"),\n"
+        "      ),\n"
+        "      website = \"http://example.com\",\n"
+        "      codeUrl = \"http://example.com\",\n"
+        "      # URL of the app's source code repository, e.g. a GitHub URL.\n"
+        "      # Required if you specify a license requiring redistributing code, but optional otherwise.\n"
+        "\n"
+        "      license = (none = void),\n"
+        "      # The license this package is distributed under.\n"
+        "\n"
+        "      categories = [],\n"
+        "      # A list of categories/genres to which this app belongs, sorted with best fit first.\n"
+        "      # See the list of categories in\n"
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
```

Comment:
Suggest: Link to https://docs.sandstorm.io/en/latest/developing/publishing-apps/